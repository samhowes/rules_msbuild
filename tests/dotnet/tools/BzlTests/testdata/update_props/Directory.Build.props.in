<Project>
    <PropertyGroup>
        <AwesomeProperty>SpecialValue</AwesomeProperty>
    </PropertyGroup>
</Project>