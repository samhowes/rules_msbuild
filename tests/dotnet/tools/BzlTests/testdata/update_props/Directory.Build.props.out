<Project>
    <PropertyGroup>
        <AwesomeProperty>SpecialValue</AwesomeProperty>
    </PropertyGroup>

    <!--  bzl:generated start  -->
    FAKE_DIRECTORY_PROPS
    <!--  bzl:generated end  -->

</Project>